### DISCUSSION TITLE
How to pass publisher bridging parameter to makeappx with packaging project?

### DISCUSSION THREAD

**Discussion by Marv51** [microsoft/WindowsAppSDK#2659] [Q&A]
Unfortunately our new code signing certificate has some changes compared to our old one. So to keep the same identity I am following this help page on: [MSIX persistent identity ](https://docs.microsoft.com/en-us/windows/msix/package/persistent-identity).

The page manually calls makeappx with a `/pb` or `/publisherBridging` parameter.

Now I am using WinAppSDK and WinUI 3 so my packaging is done with the packaging project (.wapproj). 

Can I pass an msbuild parameter to set that makeappx parameter somehow?
If not, do you have any instructions available how to manually create an msixbundle from my WinAppSDK/WinUI 3 project?

**Comment by kpotterams** [comment]
I'm having the same problem... it doesn't help that there's absolutely no documentation on the various appx parameters for msbuild :-\

**Comment by yair100** [comment]
@Marv51 did you end up trying the `publisherBridging` arg with msbuild?

**Reply by Marv51** [reply]
Publisher Bridge does not work for our requirements, as it only works on Windows 11.

That is among many issues the deal breaker for us. 

I filed/documented some more issues here: https://github.com/MicrosoftDocs/msix-docs/issues/326

**Comment by whiskhub** [comment]
A solution was posted here - cross-referencing it here so it's easier to find: https://github.com/MicrosoftDocs/msix-docs/issues/326#issuecomment-1167956865

```xml
<PropertyGroup>
   <CreatePackageMakeAppxParameters>/pb "C:/[full-path-worked-for-me]/certificate-migration-2022-pb.txt"</CreatePackageMakeAppxParameters>
   <CreateResourcePackageMakeAppxParameters>/pb "C:\[full-path-worked-for-me]\certificate-migration-2022-pb.txt"</CreateResourcePackageMakeAppxParameters>
   <AppxPackageSigningEnabled>false</AppxPackageSigningEnabled>
</PropertyGroup>
```
